src/telemetry/telemetry.c

```diff
@@ -120,13 +87,12 @@ telemetry_parse_version(const char *json, const long installed_version[3], Versi
 static void
 process_response(const char *json)
 {
-	const long	installed_version[3] = {
-		strtol(TIMESCALEDB_MAJOR_VERSION, NULL, 10),
-		strtol(TIMESCALEDB_MINOR_VERSION, NULL, 10),
-	strtol(TIMESCALEDB_PATCH_VERSION, NULL, 10)};
-	VersionResult result = {0};
+	VersionInfo installed_version;
+	VersionResult result;
+
+	version_get_info(&installed_version);
 
-	if (!telemetry_parse_version(json, installed_version, &result))
+	if (!telemetry_parse_version(json, &installed_version, &result))
 	{
 		if (NULL == result.versionstr)
 			elog(ERROR, "could not get TimescaleDB version from server response");
```

Comment:
Maybe here and generally in the telemetry BGW, instead of error-ing out, do we want to print a WARNING instead? Because it is otherwise confusing to users who see this ERROR in their logs (when in fact, the ERROR does not affect the normal functioning of the main DB).